sys/Kconfig

```
@@ -58,6 +58,16 @@ rsource "usb/Kconfig"
 rsource "xtimer/Kconfig"
 rsource "ztimer/Kconfig"
 
+config MODULE_CPP
+    bool
+    depends on TEST_KCONFIG
+    depends on HAS_CPP
+
+config MODULE_LIBSTDCPP
+    bool
+    depends on TEST_KCONFIG
+    depends on HAS_LIBSTDCPP
+
```

I think these should be visible, and enabling libstdc++ depends on having c++ enabled:

```suggestion
menuconfig MODULE_CPP
    bool "C++ support"
    depends on TEST_KCONFIG
    depends on HAS_CPP

config MODULE_LIBSTDCPP
    bool "libstdc++ support"
    depends on MODULE_CPP
    depends on HAS_LIBSTDCPP

```